sdk/containerregistry/azure-containers-containerregistry/src/main/java/com/azure/containers/containerregistry/implementation/models/Annotations.java

```diff
@@ -0,0 +1,374 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.containers.containerregistry.implementation.models;
+
+import com.azure.core.annotation.Fluent;
+import com.fasterxml.jackson.annotation.JsonAnyGetter;
+import com.fasterxml.jackson.annotation.JsonAnySetter;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.time.OffsetDateTime;
+import java.util.HashMap;
+import java.util.Map;
+
+/** Additional information provided through arbitrary metadata. */
+@Fluent
+public final class Annotations {
+    /*
+     * Date and time on which the image was built (string, date-time as defined
+     * by https://tools.ietf.org/html/rfc3339#section-5.6)
+     */
+    @JsonProperty(value = "org.opencontainers.image.created")
```

Comment:
@jianghaolu, @srnagar, do you know why this didn't generate with `JsonFlatten`?

Reply:
Let me know if I am missing any parameters to the swagger. The auto-rest help shows JsonFlatten as a customization - [https://github.com/Azure/autorest.java#add-an-annotation-to-a-class](https://github.com/Azure/autorest.java#add-an-annotation-to-a-class)

---
In reply to: [599179384](https://github.com/Azure/azure-sdk-for-java/pull/20039#discussion_r599179384) [](ancestors = 599179384)